crates/aide/src/axum/mod.rs

```diff
@@ -863,7 +867,7 @@ mod tests {
     #[test]
     fn test_nesting_with_nondefault_state() {
         let _app: ApiRouter = ApiRouter::new()
-            .nest_api_service("/", ApiRouter::new().with_state(1_isize))
+            .nest_api_service("/home", ApiRouter::new().with_state(1_isize))
```

Comment:
why is the change here needed?

Reply:
<img width="800" alt="Bildschirmfoto 2025-01-03 um 03 54 46" src="https://github.com/user-attachments/assets/c2388e15-7428-409b-9398-5dfe889fd1b7" />